.env.test

```diff
@@ -0,0 +1,13 @@
+# Sample .env file
```

Comment:
As a heads up, we already have `.env.sample`, so if you need to push up any changes you can toss this file.